### ISSUE TITLE
[package] etl/20.32.1: ERROR: : 'settings.compiler.cppstd' value not defined

### ISSUE THREAD

**Issue by Mipsters** [conan-io/conan-center-index#12589]
### Package and Environment Details

* Package Name/Version: **etl/20.32.1**
* Operating System+version: **Windows 11**
* Compiler+version: **MSVS 2022**
* Docker image: **None**
* Conan version: **conan 1.51.3**
* Python version: **Python 3.10.5**

### Conan profile

Configuration for profile default:

[settings]
os=Windows
os_build=Windows
arch=x86_64
arch_build=x86_64
compiler=msvc
compiler.version=193
build_type=Release
[options]
[conf]
[build_requires]
[env]

### Steps to reproduce

```
conan install etl/20.32.1@_/_
```

### Logs

<details><summary>Click to expand log</summary>

```
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=msvc
compiler.runtime_type=Release
compiler.version=193
os=Windows
os_build=Windows
[options]
[build_requires]
[env]

etl/20.32.1: Not found in local cache, looking in remotes...
etl/20.32.1: Trying with 'conancenter'...
Downloading conanmanifest.txt completed [0.10k]
Downloading conanfile.py completed [2.52k]
Downloading conan_export.tgz completed [0.25k]
Decompressing conan_export.tgz completed [0.00k]
etl/20.32.1: Downloaded recipe revision 0
ERROR: etl/20.32.1: 'settings.compiler.cppstd' value not defined
```

</details>

**Comment by uilianries** [comment]
@Mipsters yes, it's a limitation which we found using msvc. We will need to fix the recipe, meanwhile, you could use "Visual Studio" as compiler.

**Comment by uilianries** [comment]
@Mipsters Doing a quick look in [settings](https://docs.conan.io/en/latest/reference/config_files/settings.yml.html?highlight=msvc) documentation, the `cppstd` is mandatory for `msvc` profiles. So you have to declared a default cppstd. However, Conan Center does not provide packages built using `cppstd`, neither with `msvc` as compiler, so you will need to build from sources. On the other hand, `Visual Studio` is available as pre-built package.

To install and build:

    conan profile update settings.compiler.cppstd=17 default
    conan install etl/20.32.1@ -pr:h=default -pr:b=default --build=missing

The first line tells to update your profile named as `default`. I guess that's your profile name, but you should fix according.

Second, you install etl normally, but prefers dual profile, as it will be the only option for Conan 2.0
